extensions/microsoft-authentication/src/AADHelper.ts

```diff
@@ -71,6 +72,12 @@ export interface ITokenResponse {
 	id_token?: string;
 }
 
+type MicrosoftTokens = [accessToken: string, idToken?: string];
+
+export interface MicrosoftAuthenticationSession extends vscode.AuthenticationSession {
```

Comment:
I think we should add a new `api.d.ts` file in the `src` with this in it, to make it a little easier for extension authors to find and copy the typings